_data/cryptocurrencies.yml

```
@@ -163,7 +163,7 @@ websites:
     img: btcmarkets.png
     tfa:
       - totp
-    doc: https://www.btcmarkets.net/faq#q27
+    doc: https://www.btcmarkets.net/frequently-asked-questions#what-is-two-factor-authentication-2fa
```

```suggestion
    doc: https://www.btcmarkets.net/frequently-asked-questions
```
The direct link does not work for me, as it's not a regular HTML anchor.